app/kuma-dp/pkg/dataplane/accesslogs/server.go

```diff
@@ -33,10 +33,23 @@ func (s *accessLogServer) NeedLeaderElection() bool {
 }
 
 func NewAccessLogServer(dataplane kumadp.Dataplane) *accessLogServer {
+
+	// Log an explicit error when access log address is too long
+	// see: https://man7.org/linux/man-pages/man7/unix.7.html
+	// see: issue #853
```

Comment:
Please remove the reference to the issue here. The best way to track this will be through following the git history of this particular piece of code.

Reply:
Done :)